backend/tracim_backend/lib/core/workspace.py

```diff
@@ -218,16 +218,40 @@ def get_one(self, workspace_id: int) -> Workspace:
                 "workspace {} does not exist or not visible for user".format(workspace_id)
             ) from exc
 
-    def get_one_by_label(self, label: str) -> Workspace:
+    def get_one_by_filemanager_filename(
```

Comment:
As filemanager filename is a webdav specific part I would have left the handler of the extension in webdav specific code and only call `get_one_by_label`.

Reply:
not really sure about this one. it's seems to me more clear to have this into core api.